### ISSUE TITLE
Profile order/sorting in the Profile Select drop down

### ISSUE THREAD

**Issue by sonofkraken** [FreeTubeApp/FreeTube#987]
Absolutely love this program and use it every day. Just have a request for a tweak that would help with the overall user experience.

**Is your feature request related to a usage problem (not a bug)? Please describe.**
I use many different profiles to keep channels organized by subject. When creating a new profile it always gets placed second in the Profile Select drop down list just under All Channels.

**Describe the solution you'd like to see implemented**
The solution I'm suggesting is to be able to reorder or sort the profiles in the Profile Select drop down list so the ones I use most often are near the top.  Perhaps this reordering could be done in the Profile Manager.

**Describe alternatives you've considered**
Another option would be to increase the container size of the drop down menu so it shows all of the profiles without having to scroll through them. This would eliminate one additional step needed to reach the desired profile and would help with UX. I feel like either of these would solve the issue though combining the two could provide even more benefits.

Thanks again for a great app!

**Comment by ClemGlacee** [comment]
Yes! That'd be very nice to have every profile shown when clicking on the top right menu or to be able to reorder those at least

**Comment by git-hop22** [comment]
Is there a way to upvote this ? This feature would be greatly appreciated!

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Wont be added
